src/core/decode_stream.js

```diff
@@ -22,6 +22,28 @@ import { Stream } from "./stream.js";
 // buffer.
 const emptyBuffer = new Uint8Array(0);
 
+function makeLengthBigEnough(startLength, minLength) {
```

Comment:
As far as I can tell this function (still) won't handle `minLength === undefined` correctly, since `makeLengthBigEnough(512, undefined) === 2199023255552`; so please add the following check to ensure that `0/null/undefined` is handled consistently:
```javascript
if (!minLength) {
  return startLength;
}
```

Reply:
*This is most likely getting into the territory micro-optimizations, but I still want to ask:*
Given that the `!minLength` should be somewhat common, considering the "streams with multiple filter entries"-case, is it not potentially a little bit more efficient with an explicit fast-path [like suggested above](https://github.com/mozilla/pdf.js/pull/15550/files#r990618696) to avoid "useless" computations and comparisons?